javascript/javascript2/week1/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Spice Trails</title>
+  </head>
+  <body>
+    <header class="header">
+      <img
+        class="logo"
```

Comment:
This `<img>` tag is missing an `alt` attribute. You can add alt="Spice Trails Logo" to improve accessibility and SEO 🌈